linux_os/guide/system/accounts/accounts-restrictions/root_logins/no_direct_root_logins/ignition/shared.yml

```diff
@@ -1,10 +1,6 @@
 # platform = multi_platform_all
```

Comment:
..but the `no_direct_root_logins` uses `multi_platform_all`. Which of them is correct?

Reply:
Oh please ignore me, the bash remediations use the same `platform` definitions.